src/components/tabs/index.tsx

```diff
@@ -78,9 +78,9 @@ const Tabs = ({
         const { content, tabId, panelId, isActive } = tabItem
         return (
           <div
-            className={s.tabPanel}
             aria-hidden={!isActive}
             aria-labelledby={tabId}
+            className={s.tabPanel}
```

Comment:
🗒️ This is just an alphabetizing change that was made as I worked through this PR